index.html

```diff
@@ -30,13 +32,27 @@ <h1 class="title">
             <p class="connect-link">
                 LET'S CONNECT
             </p>
-            <div class="social-media">
-                <img src="images/icons/twitter.svg" alt="Twitter icon">
-                <img src="images/icons/linkedin.svg" alt="Linkedin icon">
-                <img src="images/icons/gmail.svg" alt="Gmail icon">
-                <img src="images/icons/github.svg" alt="Github icon">
-                <img src="images/icons/peace.svg" alt="Peace icon">
-            </div>
+            <ul class="social-media">
+                <li>
+                    <a href="https://twitter.com/PhoneKhaingHein" target="_blank"><img src="images/icons/twitter.svg"
+                            alt="Twitter icon"></a>
+                </li>
+                <li>
+                    <a href="https://www.linkedin.com/in/phone-khaing-hein-759497216/" target="_blank"><img
+                            src="images/icons/linkedin.svg" alt="Linkedin icon"></a>
+                </li>
+                <li>
+                    <a href="mailto:pkh2662003@gmail.com" target="_blank"><img src="images/icons/gmail.svg"
+                            alt="Gmail icon"></a>
+                </li>
+                <li>
+                    <a href="https://github.com/Rimuru-Tempest-26" target="_blank"><img src="images/icons/github.svg"
+                            alt="Github icon"></a>
+                </li>
+                <li>
+                    <img src="images/icons/peace.svg" alt="Peace icon">
+                </li>
+            </ul>
```

Comment:
- [ ] Kindly check your social media icons links because its not working check below:

Your Design:
![Screenshot from 2023-02-18 01-45-39](https://user-images.githubusercontent.com/45226848/219790523-30827c6f-9c90-4d0d-a8dc-ff6c10a174a0.png)

Figma design:
![Screenshot from 2023-02-18 01-45-56](https://user-images.githubusercontent.com/45226848/219790565-b500437d-3b0a-4132-9cab-df2dce7cac6c.png)

Reply:
Can you please tell me if the V-shaped two fingers represent which social media?

Reply:
![Screenshot 2023-02-18 023929](https://user-images.githubusercontent.com/85481456/219795459-eb8ad83b-896b-4c5e-ada0-65e6f4b7c6f5.png)

On my computer, they work correctly. You can check the demo link too https://phonekhainghein-portfolio.netlify.app/